testing/util.py

```diff
@@ -67,7 +67,7 @@ def broken_deep_listdir():  # pragma: no cover (platform specific)
 
 
 def platform_supports_pcre():
-    output = cmd_output(GREP, '-P', "name='pre'", 'setup.py', retcode=None)
```

Comment:
oh and remove the `name=` part too :)

Reply:
Ah yes, of course - thanks!

Reply:
oh uh right, and the line below it needs updating too to validate the output, looks like any of these will work:

```console
$ grep -P "Don't" CHANGELOG.md 
- Don't filter `GIT_SSH_COMMAND` env variable from `git` commands
- Don't filter `GIT_EXEC_PREFIX` env variable from `git` commands
- Don't write ANSI colors on windows when color enabling fails
- Don't crash when `always_run` is `True` and `files` is not provided.
- Don't run on staged deleted files that still exist
- Don't rename across devices when creating sqlite database
- Don't use readlink -f
```

I'd do the last one since it's the shortest :)

Reply:
Appreciate the help as I'm learning a lot in the process :-)

Reply:
No problem :) you can also run these tests locally as well -- the contributing guide goes into a little bit of detail there: https://github.com/pre-commit/pre-commit/blob/master/CONTRIBUTING.md